test/parallel/test-eventtarget.js

```diff
@@ -479,3 +479,12 @@ let asyncTest = Promise.resolve();
     throws(() => eventTarget.addEventListener('foo', Symbol()), TypeError);
   });
 }
+{
+  const eventTarget = new EventTarget();
+  const event = new Event('foo');
+  eventTarget.addEventListener('foo', (e) => {
```

Comment:
This test isn't quite right - the case fixed was "what happens when you dispatch an event with no listeners on an event target". The bug was "its .target isn't set".

I think a more appropriate test would be:

```js
const ee = new EventTarget();
const e = new Event('foo');
ee.dispatchEvent(e);
strictEqual(e.target, ee);
```

Reply:
Updated the test case to cover `e.target` test without listeners. Thank you!